app/frontend/src/locales/fr/translation.json

```
@@ -81,6 +81,10 @@
         "minimumSearchScore": "Score de recherche minimum",
         "minimumRerankerScore": "Score minimum du reclasseur sémantique",
         "retrieveCount": "Récupérer ce nombre de résultats de recherche :",
+        "includeCategory": "Inclure la catégorie",
+        "includeCategoryOptions": {
+            "all": "Tous"
```

Is "all" referring to categories? If that the case, please accept the suggestion.
```suggestion
            "all": "Toutes"
```